src/GitExtensions.GerritPlugin/GerritUtil.cs

```diff
@@ -27,6 +28,26 @@ internal static class GerritUtil
             var fetchUrl = GetFetchUrl(module, remote);
 
             return await RunGerritCommandAsync(owner, module, command, fetchUrl, remote, stdIn).ConfigureAwait(false);
+        }
+
+        public static bool HadNewChange(string commandPrompt, out string changeUri)

```

Comment:
Assert that `commandPrompt` isn't null or empty.

Reply:
👍